manifests/fluentd/fluentd-cluster-cfg-output-buffer-example.yaml

```diff
@@ -63,7 +63,7 @@ spec:
       type: file
       path: /buffers/stdout.log
   - elasticsearch:
-      host: elasticsearch-logging-data.kubesphere-logging-system.svc
+      host: elasticsearch-logging-data.fluent.svc
```

Comment:
I think this part no need to change? It will use in the ks-installer？

Reply:
> I think this part no need to change? It will use in the ks-installer？

Or just leave it empty like <elastic-cluster-host>

Reply:
The configuration of `ks-installer` has nothing to do with the configuration of this repository. I think the files in the `manifest` folder should be enabled as soon as they are deployed like the original.

Reply:
Keep it same to  https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator/values.yaml#L98

Reply:
> Keep it same to https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator/values.yaml#L98

thanks,fixed.